catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/source/impl/SourceDescriptorImpl.java

```diff
@@ -32,15 +34,19 @@
 
   protected Date lastAvailableDate = null;
 
+  private List<Action> actions;
+
   /**
    * Instantiates a new SourceDescriptorImpl.
    *
    * @param sourceId the source's id
    * @param catalogedTypes the cataloged types
    */
-  public SourceDescriptorImpl(String sourceId, Set<ContentType> catalogedTypes) {
+  public SourceDescriptorImpl(
```

Comment:
Are `null` checks not necessary here?

Reply:
There doesn't seem to be clear a clear project standard on argument null-checking. I believe that null-checks should only be used in cases where the cause of an NPE is non-obvious.